v2/internal/controllers/recordings/Test_KeyVault_Vault_CRUD.yaml

```diff
@@ -66,21 +66,101 @@ interactions:
     code: 200
     duration: ""
 - request:
-    body: '{"location":"westus2","name":"asotest-keyvault-ngmgjs","properties":{"accessPolicies":[{"applicationId":"00000000-0000-0000-0000-000000000000","objectId":"00000000-0000-0000-0000-000000000000","permissions":{"certificates":["get"],"keys":["get"],"secrets":["get"],"storage":["get"]},"tenantId":"00000000-0000-0000-0000-000000000000"}],"enableSoftDelete":false,"sku":{"family":"A","name":"standard"},"tenantId":"00000000-0000-0000-0000-000000000000"}}'
+    body: ""
```

Comment:
Is this payload change expected?

Reply:
It's for a different URL than before (though it doesn't look it, if you see down on line 76 the URL is new), so I think the answer is yes.

GET shouldn't have a request body, so this is right.

This is probably because I changed some of the KV-using tests to purge+create so that they didn't need a manual step to start.